src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -64,14 +59,23 @@ public NsUser getWriter() {
         return writer;
     }
 
-    public String getContents() {
-        return contents;
-    }
-
     public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public DeleteHistory deleteAnswer(NsUser loginUser, LocalDateTime deleteTime) {
+        validateDelete(loginUser);
+        this.deleted = true;
+
+        return DeleteHistory.createAnswerHistory(this.id, loginUser, deleteTime);
```

Comment:
```suggestion
        return DeleteHistory.answerOf(this.id, loginUser, deleteTime);
```

`createAnswerHistory`가 문제가 있는것은 아니라고 생각합니다.

하지만 저의 관점으로는,
`create`는 새로운 리턴객체 타입으로 추정이 가능할것같구,
`history`는 이미 객체이름이 `DeleteHistory`이기 때문에 뺄것 같습니다. 😄

Reply:
좋은거 같습니다 정적팩토리 메소드 이름 규칙에 부합하겠네용!